Add meta descriptions to Lisk-SDK, closes #371 (#387)
LiskHQ/lisk-docs: modules/ROOT/pages/lisk-elements/packages/api-client.adoc

= Lisk Elements API Client
Mona Bärenfänger <mona@lightcurve.io>

> :description: Technical reference for the API Client package of Lisk Elements. It copmrises usage examples, available options and example responses.

Reviewer: ```suggestion
:description: Technical references regarding the API Client package of Lisk Elements. This consists of usage examples, available options and example responses.
```
